snake/main.py

```diff
@@ -7,6 +7,8 @@
 from pygame import event, gfxdraw
 
 
+pygame.init()
```

Comment:
This initializes all the pygame modules, but we only need font and display for now, so you can initialize only the font. The place you put this line is also not right, you should put it where I'm already initializing the display, to keep them in the same place.